Packs/Okta/Integrations/Okta_v2/Okta_v2.py

```diff
@@ -496,6 +496,33 @@ def clear_user_sessions(self, user_id):
             resp_type='text'
         )
 
+    def get_zone(self, args, zoneID):
+        uri = f'zones/{zoneID}'
+        return self._http_request(
+            method='GET',
+            url_suffix=uri,
+            resp_type='text'
+        )
+
+    def list_zones(self, args):
+        uri = f'zones'
+        return self._http_request(
+            method='GET',
+            url_suffix=uri,
+            resp_type='text'
+        )
+
+    def update_zone(self, args, zoneObject):
```

Comment:
to Client functions you better pass not `args` but specific arg names, like 
`def update_zone(self, id, zoneObject):`